python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -354,6 +385,7 @@ def __call__(self, fn: Callable) -> AssetsDefinition:
                 tags={
                     **({"kind": self.compute_kind} if self.compute_kind else {}),
                     **(self.op_tags or {}),
+                    **code_origin_tag,
```

Comment:
to avoid this function getting huge, what do you think about making a helper function and just having this be `**get_code_origin_tags()`?